homework/hw01-tolstoy.py

```diff
@@ -0,0 +1,16 @@
+from pyspark import SparkContext
+text_file = SparkContext.textFile(FileName.txt)#https://www.gutenberg.org/files/2600/2600-0.txt
```

Comment:
`FileName.txt` will throw an error (should be in quotes) and your script should be downloading the file from the internet in the first place

Reply:
Thanks for your comment. I have done using BeautifulSoup using Jupiter Notebook for myself. However, as a learning exercise, I was trying to read on pySpark and attempted this homework using pySpark before we move on but did work much. Thanks for all.